packages/polaris-viz/src/hooks/tests/useWatchActiveSeries.test.tsx

```diff
@@ -0,0 +1,44 @@
+import {mount} from '@shopify/react-testing';
+
+import {useWatchActiveSeries} from '../useWatchActiveSeries';
+
+describe('useWatchActiveSeries', () => {
```

Comment:
- Nit: `useWatchActiveSeries()` as the describe as it's a listener
- Should have a nested `useWatchActiveSeries()` and `setActiveSeriesListener()` so that we test both

Reply:
```tsx
describe('useWatchActiveSeries()', () => {
  describe('setActiveSeriesListener()', () => {
    it('generates an event name using the given ID', () => {
      const id = getEventName('someChartId', COLOR_VISION_SINGLE_ITEM);
      const addEventListenerSpy = jest.spyOn(window, 'addEventListener');
      setActiveSeriesListener('someChartId', () => {});
      expect(addEventListenerSpy).toHaveBeenCalledWith(id, expect.any(Function));
    });
  });
  describe.todo('useWatchActiveSeries()');
});
```